src/tcgwars/logic/impl/gen8/BattleStyles.groovy

```diff
@@ -2904,7 +2728,7 @@ public enum BattleStyles implements LogicCardInfo {
         }
         onPlay {
           shuffleDeck(hand.getExcludedList(thisCard))
-          hand.removeAll(hand.getExcludedList(thisCard))
+          hand.getExcludedList(thisCard).discard()
```

Comment:
This duplicates all cards other than bruno in the player's hand.

Reply:
No it doesn't. Discard moves cards, it does not make new ones.

Reply:
Oh it's supposed to move them to deck? I'll double check this one. Using add/remove/removeAll on a card list for a proper effect should not be done though.

Reply:
moveTo followed by shuffleDeck is what I would use here.

Reply:
Agreed.